circle.yml

```diff
@@ -629,124 +597,86 @@ workflows:
           chromeVersion: "Google Chrome 88"
           firefoxVersion: "Mozilla Firefox 89"
       - build-browser-image:
-          name: "browsers node14.17.0-chrome91-ff89"
           dockerTag: "node14.17.0-chrome91-ff89"
           chromeVersion: "Google Chrome 91"
           firefoxVersion: "Mozilla Firefox 89"
       - build-browser-image:
-          name: "browsers node14.7.0-chrome84"
           dockerTag: "node14.7.0-chrome84"
           chromeVersion: "Google Chrome 84"
       - build-browser-image:
-          name: "browsers node16.5.0-chrome94-ff93"
           dockerTag: "node16.5.0-chrome94-ff93"
           chromeVersion: "Google Chrome 94"
           firefoxVersion: "Mozilla Firefox 93"
 
   build-included-images:
     jobs:
       - build-included-image:
-          name: "included 6.0.0"
```

Comment:
name is not a parameter on the build-included-image job. This value is not used.

Reply:
See my earlier comment.